recipe/meta.yaml

```diff
@@ -12,22 +12,21 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
 
 requirements:
   host:
     - python >=3.6
     - pip
```

Comment:
missing numpy dependency in host section

Reply:
Added thanks!